lib/Elastica/Param.php

```diff
@@ -138,9 +138,12 @@ public function setParams(array $params)
     public function addParam($key, $value)
     {
         if ($key != null) {
-            if (!isset($this->_params[$key])) {
-                $this->_params[$key] = array();
-            }
+            /**
```

Comment:
Its not the change thats the problem, its that there is no need to have it commented out with a TODO comment.